scheduler/app/libs/tasks/get_assets.py

```diff
@@ -126,12 +135,28 @@ def augment_total_balance(
 
 
 async def build_treasury_with_assets(
-    treasury_address: str, chain_id: int, start: str, end: str
+    treasury_address: str, chain_id: int, start: str = None, end: str = None
 ) -> tuple[Treasury, dict[str, DF], dict[str, DF]]:
     treasury = filter_out_small_assets(await make_treasury(treasury_address, chain_id))
     augmented_token_hist_prices = augment_token_hist_prices(
         await get_token_hist_prices(treasury)
     )
+
```

Comment:
I think we should not add any more code in this function, as it's a very central point of failure. Also, having default values for `start` and `end` is useful sometimes, but not always. For instance, `main.portfolio` endpoint doesn't need defaults.

Just add your date-making logic inside of `reload_treasuries_data`.

Reply:
noted, changes have been made